compaction_test.py

```diff
@@ -16,7 +16,7 @@
 logger = logging.getLogger(__name__)
 
 strategies = ['LeveledCompactionStrategy', 'SizeTieredCompactionStrategy', 'DateTieredCompactionStrategy']
-
+strategies_without_dtcs = ['LeveledCompactionStrategy', 'SizeTieredCompactionStrategy']
```

Comment:
Maybe instead of creating separate lists, and then separate tests for different versions, you would consider creating a function which returns available strategies for a given cluster version, so that the tests would stay mostly version-agnostic?

Reply:
@jacek-lewandowski I can inline this to the annotation directly, it is used in two places only.